utilities/github-sarif-helper/src/semgrep-json-to-sarif.py

```diff
@@ -224,7 +331,7 @@ def main():
 
     # Load findings from the specified file
     findings = load_findings(args.json)
-    findings['results'] = filter_findings_results(findings)
+    # findings['results'] = filter_findings_results(findings)
```

Comment:
Was this for testing or intended to stay commented? If the latter, maybe a comment explaining why ("uncomment for ...")